src/pages/eol.js

```diff
@@ -0,0 +1,214 @@
+import React, { useMemo, useState } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import { graphql } from 'gatsby';
+import {
+  Layout,
+  Icon,
+  Link,
+  useTranslation,
+} from '@newrelic/gatsby-theme-newrelic';
+import EolTable from '../components/EolTable';
+import { DIRECTION } from '../components/EolTable/TableHeader';
+import PageTitle from '../components/PageTitle';
+import SEO from '../components/SEO';
+import ErrorBoundary from '../components/ErrorBoundary';
+import { TYPES } from '../utils/constants';
+
+// @tw: add any body content here.
+// it will go under the main heading and above the EOL timeline.
+const body = <p>howdy 🤠</p>;
```

Comment:
@tabathadelane I think we want to remove this :)

Reply:
Oh right. This was requested by Austin. Can we put some content in here?

Reply:
^ I just commented this out for now.

Reply:
Oh, I didn't realize we wanted some content to go there!